frontend/dispatcher.lua

```diff
@@ -570,7 +570,7 @@ function Dispatcher:menuTextFunc(settings)
             if item == "settings" then item = next(settings, item) end
             action_name = Dispatcher:getNameFromItem(item, settings)
         else
-            action_name = T(N_("", "%1 actions", count), count)
+            action_name = T(NC_("Dispatcher", "", "%1 actions", count), count)
```

Comment:
This isn't really right either since the singular is used as the ID, but it might work as long as there's only one per context.

Reply:
It seems that english singular is missing in weblate.
![weblate_dispatcher_singular_string_missing](https://user-images.githubusercontent.com/29783895/199555352-215a5a60-ab01-4d9d-bd58-ac2bf2f4be6a.jpg)

Reply:
Given all the tools expect it plus resulting confusion it'll be best to put an unused singular in there.

The "problem" is that if there's only one action, the name of the action is shown. It only says 2 actions, 3 actions, etc. if there's more than 1.